specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/disk.json

```diff
@@ -720,6 +720,15 @@
           "type": "string",
           "description": "ARM id of the DiskAccess resource for using private endpoints on disks."
         },
+        "burstingEnabledTime": {
+          "type": "string",
```

Comment:
THis is a read only property. Should be marked as such

Reply:
Done